### PR TITLE
Updating Python docs to include the Create Environment command

### PR DESCRIPTION
**Pull request by cwebster-99**
Adding steps and pictures to the Python environments documentation to include the Create Environment command.

### CODE REVIEW

File: docs/python/environments.md [hunk 1]
```diff
@@ -63,7 +63,28 @@ The extension also loads an [environment variable definitions file](#environment
 
 ## Creating environments
 
-### Create a virtual environment
+### Using the Create Environment Command
+From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. You can also trigger the `Python: Create Environment' command through the Getting Started with Python page.
+
+The command presents a list of environment types, Venv or Conda.
```
Comment (luabud) [inline docs/python/environments.md:69]:
```suggestion
The command presents a list of environment types: **Venv** or **Conda**.
```

File: docs/python/environments.md [hunk 2]
```diff
@@ -63,7 +63,28 @@ The extension also loads an [environment variable definitions file](#environment
...
+
+![Create Environment dropdown](images/environments/create_environment_dropdown.png)
+
+If you are creating an environment using **Venv**, the command presents a list of interpreters that can be used for your project.
```
Comment (luabud) [inline docs/python/environments.md:73]:
```suggestion
If you are creating an environment using **Venv**, the command presents a list of interpreters that can be used as a base for the new virtual environment.
```

**Comment by luabud** [comment]
Looks great! If we remove the sentence around the walkthrough we can merge this any time actually :)

File: docs/python/environments.md [hunk 3]
```diff
@@ -63,7 +63,28 @@ The extension also loads an [environment variable definitions file](#environment
 
 ## Creating environments
 
-### Create a virtual environment
+### Using the Create Environment Command
+From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. You can also trigger the `Python: Create Environment' command through the Getting Started with Python page.
```
Comment (cwebster-99) [inline docs/python/environments.md:67]:
```suggestion
From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. 
```

**Review by gregvanl — APPROVED** [review]
Thanks!

**Pull request merged**